caas/kubernetes/provider/utils/labels_test.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package utils_test
+
+import (
+	"context"
+
+	jc "github.com/juju/testing/checkers"
+	gc "gopkg.in/check.v1"
+	core "k8s.io/api/core/v1"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/fake"
+
+	"github.com/juju/juju/caas/kubernetes/provider/utils"
+)
+
+type LabelSuite struct {
```

Comment:
Are there enough tests here to cover the new methods added?
(sort of related the comment elsewhere about using hand crafted maps in tests)

Reply:
Done & Done

Reply:
Wrong one. Adding some more tests now. Let that slip through.